libqtile/backend/wayland/qw/util.c

```diff
@@ -122,3 +122,48 @@ bool qw_surfaces_on_same_output(struct wlr_surface *surface_a, struct wlr_surfac
     }
     return false;
 }
+
+struct qw_view *qw_view_from_wlr_surface(struct wlr_surface *surface, bool *is_layer_surface, bool *is_session_lock_surface) {
+    is_layer_surface = false;
+    is_session_lock_surface = false;
```

Comment:
```suggestion
    *is_layer_surface = false;
    *is_session_lock_surface = false;
```
I think we need this as we're receiving pointers to those values.